examples/compose/README.md

```diff
@@ -58,14 +58,20 @@ To stop the containers and remove attached volumes, run;
 bastion/examples/compose$ docker-compose down -v
 ```
 
+## Advanced Usage
+
 ### Build from source
-To stop the containers and remove attached volumes, run;
+To build the bastion container from source files in this repo;  
```

Comment:
@jawabuu , please make all your changes to the README.yaml, then rebuild. See comments above.

Reply:
Hey @maximmi @osterman , does this apply to the README in the subdirectory?

Reply:
nope, only the root one